packages/repository/src/repositories/relation.repository.ts

```diff
@@ -67,11 +74,10 @@ export class DefaultHasManyEntityCrudRepository<
 > implements HasManyRepository<TargetEntity> {
   /**
    * Constructor of DefaultHasManyEntityCrudRepository
-   * @param targetRepositoryGetter the related target model repository instance
+   * @param getTargetRepository the related target model repository instance
```

Comment:
_the related target model repository instance_ is misleading, please fix the comment to mention a getter.